stripe-ui-core/src/main/java/com/stripe/android/uicore/image/StripeImage.kt

```diff
@@ -39,10 +41,62 @@ import kotlinx.coroutines.launch
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 @Composable
 fun StripeImage(
+    model: Any,
+    contentDescription: String?,
+    modifier: Modifier = Modifier,
+    imageLoader: StripeImageLoader = LocalContext.current.imageLoader,
+    contentScale: ContentScale = ContentScale.Fit,
+    errorContent: @Composable BoxWithConstraintsScope.() -> Unit = {},
+    loadingContent: @Composable BoxWithConstraintsScope.() -> Unit = {}
+) {
+    when (model) {
+        is String -> {
+            StripeImageWithUrl(
+                url = model,
+                contentDescription = contentDescription,
+                modifier = modifier,
+                imageLoader = imageLoader,
+                contentScale = contentScale,
+                errorContent = errorContent,
+                loadingContent = loadingContent
+            )
+        }
+        is Uri -> {
+            if (model.isRemote()) {
+                StripeImageWithUrl(
+                    url = model.toString(),
+                    contentDescription = contentDescription,
+                    modifier = modifier,
+                    imageLoader = imageLoader,
+                    contentScale = contentScale,
+                    errorContent = errorContent,
+                    loadingContent = loadingContent
+                )
+            } else {
+                Image(
+                    modifier = modifier,
+                    contentDescription = contentDescription,
+                    contentScale = contentScale,
+                    painter = rememberDrawablePainter(
+                        drawable = LocalContext.current.getDrawableFromUri(
+                            model
+                        )
+                    )
+                )
+            }
+        }
```

Comment:
can we create a StripeImageWithLocalUri `@Composable` that takes the same inputs (besides the model) as `StripeImageWithUrl` does? If we start extending the `StripeImage` component we might start having inconsistent behaviors across model types otherwise.

Reply:
as discussed offline, removed the changes in `StripeImage` will use `DrawablePainter ` with Image directly